data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -217,28 +219,8 @@ def _create_score_data(
             right_on=self.STATE_CODE_COLUMN,
             how="left",
         )
-
-        # check if there are census tracts without score
-        logger.info("Removing tract rows without score")
-
-        # merge census tracts with score
-        merged_df = national_tract_df.merge(
-            score_county_state_merged,
-            on=self.GEOID_TRACT_FIELD_NAME,
-            how="left",
-        )
-
-        # recast population to integer
-        score_county_state_merged["Total population"] = (
```

Comment:
I'm pretty sure this line was a no-op, given `score_county_state_merged` never got used in the return value --- and since we join to national_tract_df, we don't convert any other ints to float64 because of nans now.

Reply:
we also shouldn't be filling NaN with 0!